kolibri/core/notifications/api.py

```diff
@@ -483,3 +485,26 @@ def parse_attemptslog(attemptlog):
 
     if notifications:
         save_notifications(notifications)
+
+
+def batch_process_attemptlogs(attemptlog_ids):
```

Comment:
appart from the tests, I don't see any place in the code where these batch_process_xxx functions are  used. I guess there's another PR or issue were this is going to be worked, right?

Reply:
Yes - the issue this is fixing is dependent on the new morango version being merged into Kolibri, as it relies on being able to add a listener for syncing to get these updated ids.

I just wanted to get the code in first, and seems like it was useful. Will continue to chase down the Windows issues.

Reply:
Thanks!
then, ready to merge as soon as the Windows™ issues are solved